### DISCUSSION TITLE
Date issues on tables at Insights > Contributors

### DISCUSSION THREAD

**Discussion by ygouzerh** [community/community#21536] [New to GitHub]
<p>Hello everyone,</p>
<p>I have created my repository the <strong>16</strong> November 2017, and when I go on the page <strong>Insights &gt; Contributors</strong> of my project, it’s about the <strong>12</strong> and the <strong>11</strong> November 2017 : is it a problem or is it because I have misunderstood something?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/a7651a7fd62787d0a2221559a1125c2ae5e5d595.png" data-download-href="/uploads/short-url/nSQlKDG3nx6txQNlz8Z9QdFGltr.png?dl=1" title="Screenshot-2017-11-17 ygouzerh git_init_ensimag.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33725211/181065350-26cb04ec-06ac-40c1-a024-a74b5915b926.png" alt="Screenshot-2017-11-17 ygouzerh git_init_ensimag.png" data-base62-sha1="nSQlKDG3nx6txQNlz8Z9QdFGltr" width="690" height="359" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/a7651a7fd62787d0a2221559a1125c2ae5e5d595_2_690x359.png, https://aws1.discourse-cdn.com/github/optimized/1X/a7651a7fd62787d0a2221559a1125c2ae5e5d595_2_1035x538.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/a7651a7fd62787d0a2221559a1125c2ae5e5d595_2_1380x718.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/a7651a7fd62787d0a2221559a1125c2ae5e5d595_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot-2017-11-17 ygouzerh git_init_ensimag.png</span><span class="informations">1894×988 53.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Best Regards,</p>
<p>ygouzerh</p>
<p>(I’m new on GitHub, please feel free to tell if my post is wrong)</p>

**Comment by Wabri** [comment, marked as answer]
<p>I think it update week by week</p>

**Comment by ygouzerh** [comment]
<p>I think you are right! </p>
<p>The dates match with each Sunday, I will see next week what will happen!</p>
<p>But It’s still weird to have these triangles that ended the 11 and 12 November, I don’t know what I’m supposed to do with this information.</p>

**Comment by Wabri** [comment]
<p>So there is:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/97ce3ede7f257384b56056e4bbcd1e5ce6827418.png" data-download-href="/uploads/short-url/lEVYAcHanhfxolX067Az1DXdNWw.png?dl=1" title="sothere.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181065355-3dd7add9-9389-4e74-ba72-81c7cdad5e6a.png" alt="sothere.png" data-base62-sha1="lEVYAcHanhfxolX067Az1DXdNWw" width="627" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/97ce3ede7f257384b56056e4bbcd1e5ce6827418_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">sothere.png</span><span class="informations">686×547 29.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ygouzerh** [comment]
<p>Ok ! So you’re right, it is updated week by week.</p>
<p>Thanks you !</p>
